web/src/components/newsCard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+export default function NewsCard({ data }) {
+    const {
+        title,
+        description,
+        image: {
+            childImageSharp: { fluid },
+        },
+    } = data;
+
+    return (
+        <div className="bg-white h-[34rem] w-80">
+            <Img className="rounded-bl-lg" fluid={fluid} />
+            <h4 className="heading-accent font-bold mt-4 pb-10 px-4 text-brand-1 text-ts-h4">
```

Comment:
I don't think we want to skip `<h3>` here and go straight to `<h4>`. If it needs to be styled as an `<h4>` per the designs then use the tailwind classes necessary to make that work.

Reply:
@asasmith He didnt assign tags to them but font sizes that correlate with html tags. I can put it as an h3, is that what you are suggesting?